README.md

```diff
@@ -78,6 +80,10 @@ The finished web app is found in the `docs/` folder (this is so that you can eas
 
 You can test the template app at <https://emilk.github.io/eframe_template/>.
 
+### Web testing/development
+
+Open `index.html#dev` to disable caching, which makes development easier.
```

Comment:
Let's change `build_web.sh` to do this!

Reply:
Also added to start_web.sh (maybe some of the build_web.sh "open" stuff should move to start_web.sh - but at another time)